lib/src/cupertino_settings_item.dart

```diff
@@ -137,21 +140,20 @@ class CupertinoSettingsItemState extends State<CupertinoSettingsItem> {
     }
 
     rowChildren.add(
-      Expanded(
-        child: Padding(
-          padding: const EdgeInsets.only(
-            left: 15.0,
-          ),
-          child: titleSection,
+      Padding(
+        padding: const EdgeInsetsDirectional.only(
+          start: 15.0,
+          end: 15.0,
         ),
+        child: titleSection,
```

Comment:
This fixes a problem in iOS when the value is too long; it pushes the main title and actually overflows the line (you see those yellow markers at the end).